### PR TITLE
Microsoft.security updatesdk 1.3.0

### PR DESCRIPTION
**Pull request by eliagrady**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by archerzz — CHANGES_REQUESTED** [review]
@eliagrady Can you take a look at the errors in the pipeline, and fix them? As far as I can see, you have the following errors which must be fixed:
- remove blank sections in `CHANGELOG.md`, like `Bug Fixed` and `Other Changes`.
  - and update a new release date
- fix or suppress word spelling issues, as the error message indicate: `To correct false positives or learn about spell checking see: https://aka.ms/azsdk/engsys/spellcheck`
- some tests require re-recording
- some classes are being deleted, but they were not marked as `[Obsolete]` before. FYI @ArthurMa1978 
- You need to update `<Version>` in `.csproj` file: https://github.com/Azure/azure-sdk-for-net/blob/169816fb2a62b943744dcc68f17bab8e68251ebe/sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Azure.ResourceManager.SecurityCenter.csproj#L3, and probably the `<ApiCompatVersion>` too.

**Comment by eliagrady** [comment]
> @eliagrady Elia Grady FTE Can you take a look at the errors in the pipeline, and fix them? As far as I can see, you have the following errors which must be fixed:
> 
> * remove blank sections in `CHANGELOG.md`, like `Bug Fixed` and `Other Changes`.
>   
>   * and update a new release date
> * fix or suppress word spelling issues, as the error message indicate: `To correct false positives or learn about spell checking see: https://aka.ms/azsdk/engsys/spellcheck`
> * some tests require re-recording
> * some classes are being deleted, but they were not marked as `[Obsolete]` before. FYI @ArthurMa1978
>               
>               Arthur Ma
>               FTE
> * You need to update `<Version>` in `.csproj` file: https://github.com/Azure/azure-sdk-for-net/blob/169816fb2a62b943744dcc68f17bab8e68251ebe/sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Azure.ResourceManager.SecurityCenter.csproj#L3
>   , and probably the `<ApiCompatVersion>` too.

blank lines removed, release date updated
fixed some spell checks - it might need a couple of iterations or a path exclusion - not sure why the spell check is a repo-global file
re-recordings were done by @ivadim 
Version is updated to 1.3.0, ApiCompatVersion is updated to last SDK release version (1.2.0-beta.4)

**Comment by eliagrady** [comment]
@ArthurMa1978 , can you please review\merge?

**Comment by eliagrady** [comment]
kind reminder

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Pending on the deprecation approval.

**Review by ArthurMa1978 — APPROVED** [review]

**Comment by eliagrady** [comment]
not a big fan of releasing beta versions until we get all preview APIs to GA status, but I can live with that - please merge

**Review by archerzz — APPROVED** [review]

File: sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Azure.ResourceManager.SecurityCenter.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <Version>1.2.0-beta.5</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.1.0</ApiCompatVersion>
+    <ApiCompatVersion>1.2.0-beta.4</ApiCompatVersion>
```
Comment (live1206) [inline sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Azure.ResourceManager.SecurityCenter.csproj:5]: This should not be updated manually.

**Pull request merged**